### PR TITLE
add first and last name column into user table & display full name on navbar

### PR DESCRIPTION
**Pull request by LeeRenJie**
depends on #130

### CODE REVIEW

File: finance-tracker/app/assets/stylesheets/custom.css.scss [hunk 1]
```diff
@@ -16,3 +16,8 @@
   display: inline-block;
   margin-bottom: 1rem;
 }
+
+.user-name {
+  margin-top: 3.5%;
+  padding-right: 10px;
+}
```
Comment (tlkiong) [inline finance-tracker/app/assets/stylesheets/custom.css.scss:23]: What does this have to do with adding column to user table?

File: finance-tracker/app/models/user.rb [hunk 2]
```diff
@@ -5,4 +5,23 @@ class User < ApplicationRecord
...
+  end
+
+  def can_track_stock?(ticker_symbol)
+    under_stock_limit? && !stock_already_tracked?(ticker_symbol)
+  end
+
+  def full_name
+    return "#{first_name} #{last_name}" if first_name || last_name
+    "Anonymous"
+  end
```
Comment (tlkiong) [inline finance-tracker/app/models/user.rb:26]: What does this have to do with adding first and last name column into user table?

File: finance-tracker/app/views/layouts/_navigation.html.erb [hunk 3]
```diff
@@ -12,6 +12,9 @@
           </li>
         </ul>
         <ul class="navbar-nav ml-auto">
+          <li class="nav-item user-name">
+            <%= fa_icon 'user' %> <%= current_user.full_name %>
+          </li>
```
Comment (tlkiong) [inline finance-tracker/app/views/layouts/_navigation.html.erb:17]: What does this have to do with adding first and last name column into user table?

**Review by tlkiong — CHANGES_REQUESTED** [review]

File: finance-tracker/app/assets/stylesheets/custom.css.scss (hunk 1 above)
Comment (LeeRenJie) [reply finance-tracker/app/assets/stylesheets/custom.css.scss:23]: edited the PR title

File: finance-tracker/app/models/user.rb [hunk 4]
```diff
@@ -5,4 +5,21 @@ class User < ApplicationRecord
...
+    stocks.count < 10
+  end
+
+  def can_track_stock?(ticker_symbol)
+    under_stock_limit? && !stock_already_tracked?(ticker_symbol)
+  end
+
+  def full_name
+    return "#{first_name} #{last_name}" if first_name || last_name
+    "Anonymous"
```
Comment (tlkiong) [inline finance-tracker/app/models/user.rb:23]:
```suggestion
    (first_name || last_name) ? "#{first_name} #{last_name}" : "Anonymous"
```
might be easier to read. wdyt?

**Review by tlkiong — CHANGES_REQUESTED** [review]

**Review by tlkiong — APPROVED** [review]
